build.sc

```diff
@@ -134,7 +136,7 @@ object packager extends ScalaModule with Bloop.Module {
 }
 
 object `generate-reference-doc` extends SbtModule with ScalaCliScalafixModule {
-  def scalaVersion = Scala.defaultInternal
+  def scalaVersion = Scala.scala3
```

Comment:
Why not change the value of `defaultInternal` to `scala3`? (and change `Scala.defaultInternal` to `Scala.scala2` when needed)

Reply:
I wanted to do it as next step - I want to first migrate to Scala 3 and then when all is working clean up the build